recipe/patches/gh21228.patch

```diff
@@ -1,47 +0,0 @@
-From 79d170ebfa523f4a4f104d8041ad880215c5d5fa Mon Sep 17 00:00:00 2001
```

Comment:
Need to restore this patch

Reply:
Done. Not sure what is going on with the conflicts :-(